### DISCUSSION TITLE
Can't change INVENTREE_SITE_URL

### DISCUSSION THREAD

**Discussion by Shinusagi** [inventree/InvenTree#8055] [Q&A]
Hello again people, first of all thanks a lot for the previous help, I realized a lot of stuff since last week and I have Inventree up and running locally only with the standard URL... however I tried to change it and then to access from other devices on the same LAN.

I read this documentation:

https://docs.inventree.org/en/stable/start/docker_install/#custom-domain

Since it is a Docker installer I went to the .env file and changed the Site URL for the IP and sometimes the Port as well I get on the Local machine, like this:

(Commented) # Site URL - update this to match your host
INVENTREE_SITE_URL="http://inventree.localhost"
INVENTREE_SITE_URL="http://192.168.1.5"

I commented the original one so I can change it back, then to apply the change (since again it is on Docker) I perform over the directory via console:

docker compose down
(Then it stops all with the previous config)
docker compose up -d --remove-orphans
(Then it starts all with the new config)

Each time I change to ANYTHING it doesn't work, I mean, the change is working since I am not being able to access from the "inventree.localhost" neither can't I access from the IP:Port I specified.
So I put it down again, change the .env file as it was before and when starting it up I can again enter the Inventere web interface.

I have no idea what I am doing wrong, I checked in the forums and used this as well:
https://github.com/inventree/InvenTree/discussions/7402

It seemed to be working for him there, but I am doubting if something is missing to make it work like that. Do I need to get Apache installed, up and running as well? Anything I can try to check this at least? 
Thanks again for the patient, I hope this also helps others when searching as the guy from that post who helped me to at least be "sure" I wasn't messing it up as the documentation explained.

**Comment by SchrodingersGat** [comment]
`INVENTREE_SITE_URL="http:/192.168.1.5"`

Should this not be `http://192.168.1.5` (there is a `/` missing in your original value)...

**Reply by Shinusagi** [reply]
> `INVENTREE_SITE_URL="http:/192.168.1.5"`
> 
> Should this not be `http://192.168.1.5` (there is a `/` missing in your original value)...

Hello, sorry it was a typo on the post, since I didn't copy it from the original computer it is installed, I re-wrote it to post on other device and missed one slash when doing it.

I tried setting a random name instead "inventree.localhost", also the machine IP alone, the machine IP with the http, the machine IP with different ports ":8000" or "5000", and still every time I change it I am not able to access from the default url but neither can I see the page from the new one set. When I set it back at how it was default I can again enter with no issue, so the change is working on the file but something is missing and I have no idea what it is.

**Comment by Shinusagi** [comment]
Ok, reading everything again I realized the guy on that previous thread first changed on the "Caddyfile" and still didn't work, so he was told to change the .env file, however I did the opposite and still didn't work, so... I changed on the "Caddyfile" as well as on ".env" on the same sections as he did and should be changed which are:

**.env File**:

```
# Site URL - update this to match your host
# INVENTREE_SITE_URL="http://inventree.localhost/"
INVENTREE_SITE_URL="http://192.168.1.5/"
```

**Caddyfile**:


```
# Change the host to your domain (this will serve at inventree.localhost)
#{$INVENTREE_SITE_URL:inventree.localhost} {
{$INVENTREE_SITE_URL:192.168.1.5} {
	import log_common inventree
```

No ports or anything needed, and now it works!! And I can access also from the LAN without even setting a specific port.
However... in middle of this I installed Apache2 and started testing some stuff, there is a config needed since if Docker is Up then Apache2 won't be able to start, and if Apache2 is already running then Docker won't be able to initialize. Since at the end I didn't really needed Apache2 I just ignored that, left Apache2 alone and focused on the Docker Container for EvenTree!

Hope this helps others with the same issue, thanks to whoever read!
